### PR TITLE
Errors Logs for Wallet Implemented

### PR DESCRIPTION
**Pull request by muktanshumishra24**
Resolves  [AR-7246](...).

## Checklist

- [x] The changes have been tested locally.

[AR-7246]: https://team-1624093970686.atlassian.net/browse/AR-7246?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by shrinathprabhu — APPROVED** [review]

File: src/utils/content.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
+const content = {
+  RPC: {
+    INPUT_EXISTS: (rpcUrl: string) =>
+      `RPC URL - ${rpcUrl} already exists, please use different one.`,
```
Comment (shaloo) [inline src/utils/content.ts:4]:
```suggestion
`RPC URL - ${rpcUrl} already exists, please use a different one.`,

File: src/utils/content.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  },
+  CHAIN_ID: {
+    INPUT_EXISTS: (chainId: string) =>
+      `Chain ID - ${chainId} already exists, please use different one.`,
```
Comment (shaloo) [inline src/utils/content.ts:8]:
```suggestion
      `Chain ID - ${chainId} already exists, please use a different one.`,

File: src/utils/content.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  },
+  NETWORK: {
+    INPUT_EXISTS:
+      'This network is current selected, please chose a different one and try again.',
```
Comment (shaloo) [inline src/utils/content.ts:12]:
```suggestion
      'This network is currently selected, please choose a different one and try again.',

Comment (shaloo) [inline src/utils/content.ts:12]:
```suggestion
      'This network is currently selected, please choose a different one and try again.',

File: src/utils/content.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  },
+  PIN: 'Enter the pin to continue.',
+  WALLET: {
+    COPY: 'Wallet address copied.',
+    INVALID: 'Please enter a valid wallet address.',
+  },
+  RECOVERY: {
+    ANSWER_ALL: 'Answer all the questions to recover key.',
```
Comment (shaloo) [inline src/utils/content.ts:20]:
```suggestion
      ANSWER_ALL: 'Answer all the questions to recover the key.',

File: src/utils/content.ts [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    QUESTIONS: 'Questions must be different!',
```
Comment (shaloo) [inline src/utils/content.ts:21]:
```suggestion
    QUESTIONS: 'Questions must be unique!',

File: src/utils/content.ts [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+  },
+  GAS: {
+    PROVIDE: 'Please provide Gas Fee.',
+    INSUFFICIENT: 'Insufficient balance to pay for gas fee.',
```
Comment (shaloo) [inline src/utils/content.ts:25]:
```suggestion
    INSUFFICIENT: 'Insufficient balance to pay for the gas fee.',

File: src/utils/content.ts [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    ESTIMATE: 'Cannot estimate gas fee. Please try again later.',
+  },
+  DETAILS: 'Enter all the details to continue.',
+  TOKEN: {
+    ADDED: 'Token Added successfully.',
+    EXISTS: 'Token already added.',
+    ETH_MAINNET: 'Token belongs to Ethereum Mainnet.',
+    SENT: 'Token sent successfully.',
+    INSUFFICIENT: 'Insufficient balance for Transfer.',
```
Comment (shaloo) [inline src/utils/content.ts:34]:
```suggestion
      INSUFFICIENT: 'Insufficient balance for transfer.',

File: src/utils/content.ts [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+    AMOUNT: 'Amount should not be greater than Max Balance.',
```
Comment (shaloo) [inline src/utils/content.ts:35]:
```suggestion
    AMOUNT: 'Amount cannot be greater than the maximum available balance.',

File: src/utils/content.ts [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+  },
+  CONTRACT: {
+    INVALID: 'Invalid contract address.',
+  },
+  MFA: {
+    SETUP: 'MFA setup completed.',
+    INCORRECT_ANSWERS: 'Incorrect answers.',
+    INCORRECT_PIN: 'Incorrect pin.',
+    QUESTIONNAIRE: 'Please fill in all the questionnaires.',
+    QUESTIONS_REPEAT: 'Questions should not be repeated.',
```
Comment (shaloo) [inline src/utils/content.ts:54]:
```suggestion
    QUESTIONS_REPEAT: 'Questions cannot be repeated.',

File: src/utils/content.ts [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+    QUESTIONS_EMPTY: 'Questions should not be empty.',
```
Comment (shaloo) [inline src/utils/content.ts:55]:
```suggestion
    QUESTIONS_REPEAT: 'Questions cannot be empty.',

File: src/utils/content.ts [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+  RPC: {
+    ERROR: 'Incorrect combination of chain Id and RPC URL.',
+    INVALID: 'Invalid RPC URL!',
+  },
+  COPY: 'Failed to copy.',
+  WALLET: {
+    COPY: 'Failed to copy wallet address.',
+  },
+  TOKEN: {
+    INVALID: 'Invalid token ID.',
```
Comment (shaloo) [inline src/utils/content.ts:69]:
```suggestion
    INVALID: 'Invalid token Id',

**Review by shaloo — CHANGES_REQUESTED** [review]
Hi @shrinathprabhu and @muktanshumishra24 some minor typos are corrected, hope this helps.

**Comment by muktanshumishra24** [comment]
Just committed all the changes, @shaloo 
You can approve if this looks good.

**Review by shaloo — APPROVED** [review]

**Pull request merged**